client/rest-high-level/src/main/java/org/elasticsearch/client/security/DeleteRoleMappingRequest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.security;
+
+import org.elasticsearch.client.Validatable;
+import org.elasticsearch.common.Nullable;
+
+import java.util.Objects;
+
+/**
+ * Request object to delete a role mapping.
+ */
+public final class DeleteRoleMappingRequest implements Validatable {
+    private final String name;
+    private final RefreshPolicy refreshPolicy;
+
+    public DeleteRoleMappingRequest(final String name, @Nullable final RefreshPolicy refreshPolicy) {
+        this.name = Objects.requireNonNull(name, "role-mapping name is missing");
```

Comment:
I'd change the message to `role mapping name is required` for consistency with other mandatory fields in API requests